render/wlr_renderer.c

```diff
@@ -10,11 +10,11 @@
 #include <wlr/config.h>
 
 #if WLR_HAS_GLES2_RENDERER
-#include <wlr/render/egl.h>
```

Comment:
Changes to this file seem unnecessary.

Reply:
Seems like this came from a rebase, I dropped the changes.

Reply:
The changes seem to still be there

Reply:
Should be gone for real now :) I think I got lost in rebase.